index.html

```diff
@@ -1,286 +1,430 @@
+          to chat please get in touch.
+        </h2>
+        <form action="https://formspree.io/f/mwkakrqj" method="POST">
+          <input
+            type="text"
+            name="fullname"
+            id="fullname"
+            placeholder="Full name"
+            maxlength="30"
+            required
+          />
+          <div class="name-form">
+            <input
+              type="text"
+              name="name"
+              id="name"
+              placeholder="First name"
+              maxlength="30"
+              required
+            />
+            <input
+              type="text"
+              name="surname"
+              id="surname"
+              placeholder="Last name"
+              maxlength="30"
+              required
+            />
+          </div>
+          <input
+            type="email"
+            id="email"
+            name="_replyto"
+            placeholder="Email address"
+            required
+          />
+          <textarea
+            name="message"
+            placeholder="Enter text here"
+            maxlength="500"
+            required
+          ></textarea>
+          <button type="submit" class="link centered-link">Send</button>
+        </form>
+      </div>
+    </main>
+    <footer class="social footer-social">
+      <a
+        class="social-link"
+        href="https://github.com/SpaYco"
+        target="_blank"
+        rel="noreferrer noopener"
+      >
+        <img
+          class="social-icon"
+          src="./media/svg/github.svg"
+          alt="GitHub Icon"
+        />
+      </a>
+      <a
+        class="social-link"
+        href="https://www.linkedin.com/in/spayco"
+        target="_blank"
+        rel="noreferrer noopener"
+      >
+        <img
+          class="social-icon"
+          src="./media/svg/linkedin.svg"
+          alt="LinkedIn Icon"
+        />
+      </a>
+      <a
+        class="social-link"
+        href="https://angel.co/spayco"
+        target="_blank"
+        rel="noreferrer noopener"
+      >
+        <img
+          class="social-icon"
+          src="./media/svg/angel.svg"
+          alt="AngelList Icon"
+        />
+      </a>
+      <a
+        class="social-link"
+        href="https://twitter.com/iSpaYco"
+        target="_blank"
+        rel="noreferrer noopener"
+      >
+        <img
+          class="social-icon"
+          src="./media/svg/twitter.svg"
+          alt="Twitter Icon"
+        />
+      </a>
+      <a
+        class="social-link"
+        href="https://spayco.medium.com/"
+        target="_blank"
+        rel="noreferrer noopener"
+      >
+        <img
+          class="social-icon"
+          src="./media/png/xicon.png"
+          alt="Medium Icon"
+        />
+      </a>
+    </footer>
+    <div class="nav-overlay off-display">
+      <div class="nav-mobile-container">
+        <div class="xicon-image"><img src="/media/png/xicon.png" alt="Exit Icon"></div>
```

Comment:
Nice of you to attempt to fix the issue with the broken link as requested by the previous reviewer but on the client side, I am still seeing that this issue is not yet resolved. 

Kindly resolve this broken link issue as it is necessary for providing a solid 💪  user experience.

![code rev](https://user-images.githubusercontent.com/84930355/204051496-00f7f80c-f8b4-4f81-a558-bc4192ecc57a.JPG)